src/main/scala/firrtl/passes/clocklist/ClockListTransform.scala

```diff
@@ -65,7 +56,7 @@ class ClockListTransform extends Transform {
   def passSeq(top: String, writer: Writer): Seq[Pass] =
     Seq(new ClockList(top, writer))
   def execute(state: CircuitState): CircuitState = getMyAnnotations(state) match {
-    case Seq(ClockListAnnotation(ModuleName(top, CircuitName(state.circuit.main)), out)) => 
+    case Seq(Annotation(ModuleName(top, CircuitName(state.circuit.main)), _, out)) => 
```

Comment:
For the record, it isn't necessary but you could add an unapply to ClockListAnnotation to be able to keep the same unapply.